packages/xo-web/src/xo-app/vm/tab-general.js

```diff
@@ -59,160 +59,178 @@ export default connectStore(() => {
     vgpuTypes,
     vm,
     vmTotalDiskSpace,
-  }) => (
-    <Container>
-      {/* TODO: use CSS style */}
-      <br />
-      <Row className='text-xs-center'>
-        <Col mediumSize={3}>
-          <h2>
-            <Number
-              value={vm.CPUs.number}
-              onChange={vcpus => editVm(vm, { CPUs: vcpus })}
-            />
-            x <Icon icon='cpu' size='lg' />
-          </h2>
-          <BlockLink to={`/vms/${vm.id}/stats`}>
-            {statsOverview && <CpuSparkLines data={statsOverview} />}
-          </BlockLink>
-        </Col>
-        <Col mediumSize={3}>
-          <h2 className='form-inline'>
-            <Size
-              value={defined(vm.memory.dynamic[1], null)}
-              onChange={memory => editVm(vm, { memory })}
-            />
-            &nbsp;
-            <span>
-              <Icon icon='memory' size='lg' />
-            </span>
-          </h2>
-          <BlockLink to={`/vms/${vm.id}/stats`}>
-            {statsOverview && <MemorySparkLines data={statsOverview} />}
-          </BlockLink>
-        </Col>
-        <Col mediumSize={3}>
-          <BlockLink to={`/vms/${vm.id}/network`}>
+  }) => {
+    const {
+      addresses,
+      CPUs,
+      current_operations: currentOperations,
+      id,
+      installTime,
+      memory,
+      os_version: osVersion,
+      power_state: powerState,
+      startTime,
+      tags,
+      VIFs,
```

Comment:
Why do you camelCase some but not all of them? Does Eslint allow this?

Reply:
Yes,  it allows that but I will use camelCamse for the rest.